### ISSUE TITLE
Fusion shouldn't be weaker than fission!

### ISSUE THREAD

**Issue by Ieldra2** [NillerMedDild/Enigmatica6#3112]
I am observing that, at the same burn/injection rate of the relevant fuels, A Mekanism fusion reactor generates considerably lesser amounts of steam than a fission reactor. Example: at an injection rate of 32, a fusion reactor generates 660000mb/t, while a fission reactor generates 1600000mb/t at a burn rate of 32 (this should be independent of reactor design, but just in case this is a 7x9x7 reactor with 78 cores, running at a temperature of 613K). This is highly unintuitive. In fact, personally I consider this a bug, but it may be intentional, so I'll post this as a suggestion.  
I would also be grateful for a suggestion where I should change this in the config in order to not mess up anything else.

**Comment by TemperedFool** [comment]
I'm not sure burn rates are intended to be comparable between fission and fusion, even in base Mekanism. We have also adjusted things to reduce the production of fusion - once it's set up it's self sustaining on just water as an input, so I'm not sure we'd even agree it should be more powerful than Fission.

You can see details of our changes here: https://github.com/NillerMedDild/Enigmatica6/wiki/Mekanism#fission-fusion

As to changing the configs, the most straightforward thing to do would be to use base Mekanism configs as a point of comparison. Do be aware that changing them may make any existing setups unstable, and that they'll get changed back every time you update the pack.

There's nothing in the pack other than the SPS (**yet**) that will use all that power, so I would suggest instead leaving the configs as-is.

**Comment by Ieldra2** [comment]
As for comparing fusion with fission, I think there should be some relation to real-world physics here, as is certainly intended by the mod maker. This is actually why I complain about this, rather than the total power generated, which, as you say, isn't used for much and doesn't have any resemblance to RL physics anyway. As I see it, you can adjust the total as you want, but making fusion weaker than fission, no, I won't agree that this is reasonable.

**Comment by TemperedFool** [comment]
Again, I'm not sure burn rates are comparable between the two. I believe fusion was balanced with the assumption that you'd target a burn rate of 98, and inject DT to boost the reactor. I don't really agree that fusion should just be always better.

Also, if we're comparing to IRL fusion, I think it should basically always be net negative on power production? Last I checked we're not able to make a fusion reactor that produces more power than it consumes. Let's hope that changes soon 😉

**Comment by MuteTiefling** [comment]
Game play trumps reality.

It's a simple tenant in game design that means if something is somehow worse for gameplay by being too realistic, then it should be adjusted to make for better game play.

In the case of power rebalancing, the goals were simple: attempt to reward people for more effort, and bring everything down to more reasonable levels.

Now, in the specific case of Fission and Fusion, as Tempered already mentioned, Fusion is just water. It has a fair amount of setup to get there and to build the infrastructure, but at the end of the day the only input is water, thus turning this complex machine into a set and forget style power not unlike solar. You're rewarded with very good power still (unlike solar) due to the complexity and requirements to get there, but the numbers had to be brought in line for gameplay. There's just no reason to be generating a billion FE/t from water.

Compare that to Fission, where you need a similarly heavy infrastructure, but also a heavy resource gathering infrastructure to obtain consistent uranium and fluorite. Not to mention the risk of explosions. Therefore Fission is harder to build and maintain, and rewarded accordingly with a higher potential output. Much higher. But reaching that high is... very challenging.

In actual gameplay, while Fission's top energy production is much higher than a single Fusion reactor, it is very unlikely that you'd ever be able to sustain that rate. There's just no good way to keep the fuel production going fast enough to keep pace. In contrast, Fusion can hum along at a cool 20 million FE/t without much issue. Just make sure you're producing enough fuel during the day to last the night. 

Now, let's say we do still want to bring real physics into this... Yes, _theoretically_ fusion is the most energetic and efficient source of power we currently know, second only to antimatter reactors. _Theoretically_. In reality, our best current Fusion reactors generate about 200x less energy than we put in to start them, and are not self sustaining by any stretch of the imagination. Will we get there? Probably. But it'll take time and many iterations before we get to fusion that's anywhere near what they can produce in theory.

So, how does this factor into Mek Fusion? Well... There's a whole big range out there from not self sustaining to very efficient. Consider our nerf to be moving Mek's Fusion further from optimal. Could it do better? Yes. But Steve doesn't have the tech for it.

**Comment by Ieldra2** [comment]
Well, I guess there's no reason for dragging this out. I've heard the arguments before, and I agree, but only to a degree, and the way this is handled in this pack goes beyond that. 

As for what's happening in-game, building the fission infrastructure was interesting, but also a great annoyance, not because of any logistics or resource collection - those are the usual difficulties you need to deal with in games like this - but again because of a design decision that flies in the face of common sense. So you can't store polonium in tanks but you can quite readily carry pellets around in your inventory. If you have an underground base you need to use lengthy tubing to reach the open sky or dig unpleasant holes because you can't use Entangloporters. WTF! For that reason my patience with Mekanism's fission is very limited, and I was looking forward to ending it after I had aquired enough Antimatter...

**Comment by TemperedFool** [comment]
> So you can't store polonium in tanks but you can quite readily carry pellets around in your inventory. If you have an underground base you need to use lengthy tubing to reach the open sky or dig unpleasant holes because you can't use Entangloporters.

This... is a Mekanism design decision. I get it if you don't like how they've designed Fission, but the design is quite intentional. I'll point out that rather than run a long tube, you can do what most of us do and build your fission setup away from your base, bringing in fuel via Entangolooper.

> As I'll still do I guess, since after 9 pellets (you can carry those around as well, just saying) there is no more need for power in the range of MFE/t.

Other than the antimatter-based Infinity tool recipes, and Mek armor/upgrades/whatever, this is certainly true... in normal mode. You're at the end game now - for Mek - likely time to move on to another mod.

**Comment by Ieldra2** [comment]
> Other than the antimatter-based Infinity tool recipes, and Mek armor/upgrades/whatever, this is certainly true... in normal mode. You're at the end game now - for Mek - likely time to move on to another mod.

Yeah, as for moving on....I don't trust setups that can explode. I won't be able to relax and move on until I can switch that time bomb off. And as much as I like the Mekasuit and -tool, does anyone ever go to the final tier with IF's infinity tools? I mean 10B of antimatter per tool.... That makes about 50 hours of a fission reactor running at burn rate 32.

**Comment by TemperedFool** [comment]
> That makes about 50 hours of a fission reactor running at burn rate 32.

I'm sure, which is why you're not intended to run your fusion reactor at a burn rate of 32 - push it to the max and see what kind of performance you get!

> does anyone ever go to the final tier with IF's infinity tools? I mean 10B of antimatter per tool....

Yes they do - it's one of the few end-game challenges of the pack, give the kitchen sink nature of it.

> I  hope E6E will follow suit.

So do we 😉

**Comment by Ieldra2** [comment]
Well, that is that. In my estimation fusion should be hard to achieve, but powerful and easy when it is achieved. One could reflect that by requiring antimatter in the fusion controller recipe, for instance. But all right, it's not super-important. 

@TemperedFool 
Running the SPS is not so much about power than raw materials. 10KB polonium requires 100KB fissile fuel to be burned, and that takes its time, and also you'll run into fluorite shortages even if you're willing to build max-sized reactors just for this. In general, I prefer solutions that are less about brute force, but that's a different issue.

**Comment by TemperedFool** [comment]
Yeah, I get that; the challenge is building up a resource generation chain that will do that for you. Not everyone's cup of tea.
